src/components/Item/VisualizationItem/Item.js

```diff
@@ -135,6 +152,22 @@ export class Item extends Component {
             : null;
     };
 
+    getPluginComponent = () =>
+        this.getActiveType() === CHART ? (
+            <div>
```

Comment:
Do we need these extra `<div>` elements?

Reply:
Thanks for reporting, forgot to remove these after checking some things.